node/src/chain_spec/mod.rs

```diff
@@ -115,108 +113,115 @@ pub fn parachain_genesis(
 	elections: Vec<(AccountId, Balance)>,
 	parachain_id: ParaId,
 	duster: DusterConfig,
-) -> RuntimeGenesisConfig {
-	RuntimeGenesisConfig {
-		system: SystemConfig {
-			// Add Wasm runtime to storage.
-			..Default::default()
-		},
-		session: SessionConfig {
-			keys: initial_authorities
-				.0
+) -> serde_json::Value {
+	serde_json::json!({
```

Comment:
Could we write any meaningful test for this, so next time when we change it we have some safety net? Or what ensures that it still works? Or you tested it manually?

Reply:
ChainSpec consists of 3 parts: client spac, genesis config and wasm code. `from_genesis()` returns `RuntimeGenesisConfig` type and I could't find a way how to use in tests( it's not printable, comparable,...). When I tried to work with `ChainSpec` type I had issues with the wasm, it's not build deterministically so it can produce different results every build. So we can't just compare the new one with the old one. Also the structure is a little bit different. But I tried to at least manually compare everything except wasm code and the content is the same.